include/evmc/evmc.h

```diff
@@ -187,6 +188,20 @@ struct evmc_message
      * Defined as `c` in the Yellow Paper.
      */
     evmc_address code_address;
+
+    /**
+     * The init code for creation message.
+     *
+     * Ignored unless kind is ::EVMC_CREATE3.
+     */
+    const uint8_t* init_code;
```

Comment:
EOF creation requires both `init_code` and `input_data` for execution.

Alternative approach to this could be new method in Host, that will take `evmc_message` and `init_code` as parameters.

Reply:
Yet another option could be passing initcontainer index instead of full init_code, but then Host would need to get parent container from state and parse it to get initcontainer from it. Seems inefficient.